From 23c1ff3c246337dbbe2aa4c17524030a620af66c Mon Sep 17 00:00:00 2001
From: Khoi Nguyen <khoi.nguyen.aj@renesas.com>
Date: Thu, 30 Jul 2020 13:28:30 +0700
Subject: [PATCH 454/458] arm64: dts: renesas: r8a774a3-hihope-rzg2m: Enable
 vspb and vspi

This patch enables vspb and vspi for RZ/G2M v3.0 board.

Signed-off-by: Khoi Nguyen <khoi.nguyen.aj@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774a3-hihope-rzg2m.dts | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a774a3-hihope-rzg2m.dts b/arch/arm64/boot/dts/renesas/r8a774a3-hihope-rzg2m.dts
index 8008aa8..0a67731 100644
--- a/arch/arm64/boot/dts/renesas/r8a774a3-hihope-rzg2m.dts
+++ b/arch/arm64/boot/dts/renesas/r8a774a3-hihope-rzg2m.dts
@@ -101,3 +101,11 @@
 	clock-names = "du.0", "du.1", "du.2",
 		"dclkin.0", "dclkin.1", "dclkin.2";
 };
+
+&vspb {
+	status = "okay";
+};
+
+&vspi0 {
+	status = "okay";
+};
-- 
2.7.4

